runtimes/js/encore.dev/req_meta.ts

```diff
@@ -10,6 +10,12 @@ export interface APIDesc {
 
   /** Whether the endpoint is a raw endpoint. */
   raw: boolean;
+
+  /** Whether the endpoint requires auth. */
+  requiresAuth: boolean;
+
+  /** Whether the endpoint is a is exposed. */
+  exposed: boolean;
```

Comment:
ah makes sense, I added these here for the previous implementation, so they are actually not needed, just keep them as it could be nice to have access to I thought. I'll remove this one, but should I keep auth/requiresAuth? or remove that as well?

Reply:
Yeah let's remove `exposed`, but can keep `requiresAuth`